### ISSUE TITLE
Logo and Banner for Algopedia

### ISSUE THREAD

**Issue by Kamran1819G** [Kamran1819G/Algopedia#10]
As an open-source project, we rely on the community for support and contributions. Having a well-designed logo and banner would help to establish our brand identity and attract more users to the project.

I am looking for a design that is simple, modern, and memorable. It should incorporate our project name and possibly some imagery or symbols that relate to our project's focus.

Share your work in this Issue comment section after selecting your work, we will assign this issue to you and then you can make changes in the repository.

**Comment by Uninterestedguy** [comment]
Hello Kamran,

I have been watching the Algopedia repository for a while and I like the idea. I have worked on a logo which might be suitable for the repository can I get this issue assigned to me ?

**Comment by Kamran1819G** [comment]
@Uninterestedguy Sure, I have assigned this issue to you now 😄👍

**Comment by Kamran1819G** [comment]
@Uninterestedguy share it here first before creating PR

**Comment by Uninterestedguy** [comment]
What do you think about these

![icon 2](https://github.com/Kamran1819G/Algopedia/assets/70902772/a97ddc23-540e-4f07-ad6d-148d6d91e1d8)
![icon 1](https://github.com/Kamran1819G/Algopedia/assets/70902772/197e0a21-7483-4668-aaed-8c3057981a4a)

**Comment by Uninterestedguy** [comment]
Different design

![icon 3](https://github.com/Kamran1819G/Algopedia/assets/70902772/43d7b1f5-a457-44b4-9801-f6275b640dec)

**Comment by Kamran1819G** [comment]
> What do you think about these
> 
> ![icon 2](https://user-images.githubusercontent.com/70902772/241785197-a97ddc23-540e-4f07-ad6d-148d6d91e1d8.png) 

Sorry for the late reply, This is looking good can you also make a banner with the same color and font and make PR of that, This logo and banner will be released after changing the UI of the website

**Comment by Uninterestedguy** [comment]
Yes, I will do it... But I need some time i will do it by 20th June
